solar-land/scss/layout/_header.scss

```diff
@@ -0,0 +1,64 @@
+@import "../../../shared/scss/components/button";
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 1rem 2rem;
```

Comment:
Oh, I didn't read the shared code. I want to refactor my code after I complete my project because some of our shared code may have changed. Thank you for your comment! I will follow our shared code and conventions.